Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -576,7 +579,35 @@ namespace Multiplayer
                     remoteAddress.c_str()) return;
             }
 
-            AZ_Printf("MultiplayerEditor", "Editor is listening for the editor-server...")
+            #ifdef _WIN32
+                // Find and terminate all ServerLauncher processes before launching a new one for the editor
```

Comment:
Do people manually launch servers much? Will this interact weirdly in that case?

Reply:
This helps in 2 situations. 1) I manually launch servers often and then forget to tear it down before running editor CTRL+G. 2) The Editor crashes often and leaves its rogue server running and hidden as a background process.